--- origsrc/pyclutter-1.0.0/clutter/Makefile.am	2009-08-25 05:03:43.000000000 -0500
+++ src/pyclutter-1.0.0/clutter/Makefile.am	2009-09-08 03:20:14.565031300 -0500
@@ -113,7 +113,7 @@ defs_DATA = \
 	&& rm -f gen-$*.c
 
 # delete the old submodules, if any, to avoid collisions
-install-data-local:
+install-data-hook:
 	(test -d $(DESTDIR)$(pyclutterexecdir) && \
 	 cd $(DESTDIR)$(pyclutterexecdir) && \
 	 rm -f cluttercairo.so && \
--- origsrc/pyclutter-1.0.0/docs/Makefile.am	2009-08-28 06:36:09.000000000 -0500
+++ src/pyclutter-1.0.0/docs/Makefile.am	2009-09-08 03:18:00.051726000 -0500
@@ -85,7 +85,7 @@ install-data-local:
 	fi
 
 install-data-hook:
-	@$(PYGOBJECT_FIXXREF) -i $(PYGOBJECT_PYGDOCS) $(DESTDIR)$(TARGET_DIR)
+	@$(PYTHON) $(PYGOBJECT_FIXXREF) -i $(PYGOBJECT_PYGDOCS) $(DESTDIR)$(TARGET_DIR)
 
 uninstall-local:
 	rm -rf $(DESTDIR)$(TARGET_DIR)/*
